toolchain/lex/tokenized_buffer_benchmark.cpp

```diff
@@ -268,7 +319,48 @@ auto RandomMixedSeq(int symbol_percent, int keyword_percent) -> std::string {
   }
   std::shuffle(tokens.begin(), tokens.end(), absl::BitGen());
 
-  return llvm::join(tokens, " ");
+  // Distribute the tokens across lines as well as horizontal whitespace. The
+  // goal isn't to make any one line representative of anything, but to make the
+  // rough density of different kinds of whitespace roughly representative.
+  //
+  // TODO: This is a really coarse approach that just picks a fixed number of
+  // tokens per line rather than using some distribution with this as the median
+  // or mean.
+  llvm::SmallVector<std::string> lines;
+  // First place tokens onto each line.
+  for (auto i : llvm::seq(NumTokens / options.tokens_per_line)) {
+    lines.push_back("");
+    llvm::raw_string_ostream os(lines.back());
+    // Arbitrarily indent each line by two spaces.
+    os << "  ";
+    llvm::ListSeparator sep;
```

Comment:
I assume we don't want commas between the tokens:
```suggestion
    llvm::ListSeparator sep(" ");
```